### DISCUSSION TITLE
Compose Multiplatform support

### DISCUSSION THREAD

**Discussion by N7ghtm4r3** [pushpalroy/JetLime#33] [General]
Hello!
Are there any updates about the porting to Compose Multiplatform?

**Comment by pushpalroy** [comment]
Support for KMP has been added.
Thanks for opening this Discussion space.

Requesting developers to use the new version in their multiplatform projects and give feedback!

**Comment by N7ghtm4r3** [comment]
Thank you and @pisoj for the migration! 
I have already implemented in a desktop application and it works perfectly
